### DISCUSSION TITLE
Laragon 7.0 subscriptions ??

### DISCUSSION THREAD

**Discussion by fifisave** [leokhoa/laragon#939] [General]
I donate for this wonderful tool but don't want to pay any yearly subscriptions, so far so I will find another free solution .
This tool is supposed to be always free sorry for that !!
regards

**Comment by 99839** [comment]
user laragon 6.0

**Reply by charles-prof** [reply]
what about [laragonzo](https://github.com/husnilkhatimi/laragonzo) folks !!!

**Reply by AltayTengri** [reply]
![Screenshot 2024-12-23 at 13-48-02 FAQ Laragon - portable isolated fast   powerful universal development environment for PHP Node js Python Java Go Ruby](https://github.com/user-attachments/assets/13c2e6e7-165a-448c-b332-b6445585f18a)
Here is says: "Yes, it is forever free." What happened to this statement??

**Reply by Sophist-UK** [reply]
Yes - I pointed this out [in my review](https://github.com/leokhoa/laragon/discussions/931#discussioncomment-11592877) c. a week ago, so I am surprised that it is still there. I guess that Leo's commitment to providing a commercial level of support for what is now a commercial product hasn't yet got to the point of bringing his FAQ up to date.

However, if the cost were $10 instead of $150, most existing users would probably pay, and that would be a pretty large initial income for Leo.

**Reply by charles-prof** [reply]
currently, it looks similar to sublime text

[Shareware is a type of proprietary software that is initially shared by the owner for trial use at little or no cost.](https://en.wikipedia.org/wiki/Shareware)

Sublime Text is [proprietary software](https://en.wikipedia.org/wiki/Proprietary_software), but can be downloaded for free and used as an evaluation version with no time limit. -  https://en.wikipedia.org/wiki/Sublime_Text

**Reply by Sophist-UK** [reply]
Sadly no, not like sublime.

**v6 is shareware:** You can use legally use v6 for nothing as long as you want (not just for evaluation). 

**v7 is licensed only:** In the absence of any formal license documentation stating otherwise, you can _**NOT**_ legally use v7 without a license for any purpose or for any evaluation period - no license, no usage.

**Reply by charles-prof** [reply]
speaking of free (dom), I think license is boring stuff
(sometimes annoying) just like you accept/reject cookies while using a website 

Apart from that, my eyes is on "dev env for **entrepreneurs**"

I see his clear vision for the future of laragon.
Also he replied to several comments saying he will make laragon even faster once he gets time

oh man i love his fighting spirit

Hi @leokhoa , i see we can build a bridge between dev environment and production environment

what do we call "reducing the production parity"

**Reply by Sophist-UK** [reply]
> he will make laragon even faster once he gets time

IMO he will make laragon even faster _**if**_ he _**ever**_ gets time - which will be if and when he gets enough income to quit his employment and go it alone - and that needs VOLUME of sales.

**Reply by charles-prof** [reply]
hi @Sophist-UK

you are a great critique, very intuitive and alive in conversation during laragon v6-7 transition

but my friend, you can reduce speculations, set him (@leokhoa) free, he will definitley shine like a north star

My perspective on laragon,

> When laragon came, it satisfied the current ( gen ) devs with an easy to use intuitive interface

> back then, the expectation is simple, now it is collaborative and team playing

If we support him like his family ( that we should without any expectations ), he will surprise us again

**like when he gave birth to laragon**

**Reply by Sophist-UK** [reply]
I am sorry if you have got the impression that I am not wanting Leo to succeed - because I would absolutely love to see him succeed.

But unfortunately, I don't think he is going down a path which will lead to success - for all the reasons I have explicitly mentioned in my past posts - but I think that there is a different path that will lead to success and I am trying - unsuccessfully so far - to get Leo to change paths before it becomes too late to do so (and IMO that point is rapidly approaching because goodwill is evaporating rapidly).

**Reply by charles-prof** [reply]
@Sophist-UK I am not having that impression...

anyway, i have a question for you

can you make a commercial software with open source components ?

GPL - kind of requires you to open source the code right ??

**Reply by Sophist-UK** [reply]
In general terms, most open source licenses allow you to distribute their product as part of a commercial product so long as you don't charge for the open source part - but as ever the devil is in the detail of each license. 

Depending on the license, if you modify the Open Source part you need to publish your changes - but since Laragon distributes the source for all the 3rd party components, this definitely happens already. 

But one other thing you almost always have to do is to include the attribution and license details in your own product - and if we are going to be picky, Leo / Laragon doesn't do this correctly in one or two minor places (and for a chargeable product Leo needs to pay attention to these sorts of details because they can lead to law suits).

**Comment by charles-prof** [comment]
idk about other things... but what i see is

*Laragon 7* requires license to use,... while laragon itself requires
license
